packages/neuron-wallet/src/controllers/sync-api.ts

```diff
@@ -137,6 +139,14 @@ export default class SyncApiController {
 
     const remainingBlocksToCache = bestKnownBlockNumber - cacheTipNumber
     const remainingBlocksToIndex = bestKnownBlockNumber - indexerTipNumber
+    const setAssumeValidTarget =
+      app.isPackaged &&
+      process.env.CKB_NODE_ASSUME_VALID_TARGET &&
+      process.env.CKB_NODE_ASSUME_VALID_TARGET_BLOCK_NUMBER
+        ? +process.env.CKB_NODE_ASSUME_VALID_TARGET_BLOCK_NUMBER
+        : undefined
```

Comment:
It could be clearer if the var name ends with `block number` and the `else branch` is `Number.NEGATIVE_INFINITY`

Reply:
I guess undefined is better which means it does not set this option.

Reply:
> I guess undefined is better which means it does not set this option.

Make sense